src/me/mrCookieSlime/Slimefun/api/item_transport/CargoNet.java

```diff
@@ -155,30 +155,33 @@ public void tick(final Block b) {
 
 			final Map<Integer, List<Location>> output = new HashMap<>();
 
+			Set<Location> combinedOutputNodes = outputNodes;
+			outputNodes.addAll(advancedOutputNodes);
```

Comment:
I think you wanna .addAll() to your combinedOutputNodes, right?
(I know it is the same reference but just trying to follow your train of thought)